mscore/file.cpp

```diff
@@ -1949,7 +1953,7 @@ bool MuseScore::saveAs(Score* cs_, bool saveCopy, const QString& path, const QSt
                   writeSessionFile(false);
                   }
             }
-      else if (ext == "musicxml") {
+      else if ((ext == "musicxml") || (ext == "xml")) {
```

Comment:
this line alone will allow uses to manually change musicxml to xml and make the .-o commandline option work

Reply:
Let's leave this line alone then here? Adding another preference for this probably rare thing might be not appropriate indeed.

Reply:
This line plus the other one from this commit? Or really require uses to manually change musicxml to xml? Esp. windows users, who are having difficulties with this already, not even seeing the extensions by default?

Reply:
it isn't a rare case though, has been requested quite often. Or had been explained to the users that they need to rename manually.

Reply:
Well, let's leave this commit alone then, without the preferences. Hopefully that `(outdated)` marking would be enough not to encourage its usage too much

Reply:
So you'll merge that 1st commit? Need me to resubmit this PR without the 2nd commit for that or can you just cherry-pick the 1st one?

Reply:
Yes, I can cherry-pick the commit, will do it now.

Reply:
Thanks